content/asp.net/getting-started/controllers-actions.md

```diff
@@ -1,20 +1,122 @@
 # Understanding Controllers and Actions
-by [Steve Smith](http://deviq.com/me/steve-smith)
+by [James Chambers](http://jameschambers.com)
 
 #### Sample Files
 Download a ZIP containing this tutorial's sample files:
-- [Initial Version] - Use this as a starting point when following along with the tutorial yourself
-- [Completed Version] - Includes the completed versions of all samples
+- [Completed Version (Pending)] - Includes the completed versions of all MVC tutorials
 
-See the [Issue](https://github.com/dotnet/training-tutorials/issues/60) to claim this lesson and/or view what should be included in it.
+## Controllers and Actions in ASP.NET Core
+As we talked about in the [last tutorial](mvc.md), a _controller_ is where we put the logic needed to respond to incoming requests. Each incoming request is evaluated and mapped into a specific action on a controller by virtue of its HTTP verb, path and querystring through a process called [routing](routing.md).
```

Comment:
I don't think "querystring" is a word, it should be "[query string](https://en.wikipedia.org/wiki/Query_string)".